### PR TITLE
clean: tracker validation codes

### PR DESCRIPTION
**Pull request by lucaCambi77**
- add new validation code E1313
- clean and reorder existing validation code based on https://github.com/dhis2/dhis2-core/pull/16833

This has a dependency with https://github.com/dhis2/dhis2-docs/pull/1357, so do not merge this PR

### CODE REVIEW

**Comment by muilpp** [comment]
are the added error codes used anywhere?

**Comment by lucaCambi77** [comment]
> are the added error codes used anywhere?

@muilpp No codes have been added; the ones you see have been moved around in proper order. Only E1313 should be added

File: src/developer/web-api/tracker.md [hunk 1]
```diff
@@ -1114,22 +1122,22 @@ otherwise specified.
...
-| E4066 | No data element found for item: `{0}` | |
-| E4067 | Attribute UID is missing in filter | |
-| E4068 | No tracked entity attribute found for attribute: `{0}` | |
+| E4014 | Relationship type `{0}` constraint requires a tracked entity having type `{1}` but `{2}` was found. | |
+| E4015 | Relationship: `{0}`, already exists. | |
+| E4016 | Relationship: `{0}`, do not exist. | |
+| E4017 | Relationship: `{0}`, is already deleted and cannot be modified. | |
+| E4018 | Relationship: `{0}`, linking {1}: `{2}` to {3}: `{4}` already exists. | |
+| E5000 | "{0}" `{1}` cannot be persisted because "{2}" `{3}` referenced by it cannot be persisted. | The importer can't persist a tracker object because a reference cannot be persisted. |
+| E5001 | "{0}" `{1}` cannot be deleted because "{2}" `{3}` referenced by it cannot be deleted. | The importer can't deleted a tracker object because a reference cannot be deleted. |
```
Comment (teleivo) [inline src/developer/web-api/tracker.md:1141]: > add new validation code E1313
https://github.com/dhis2/dhis2-docs/pull/1354#issuecomment-2014706880 you said only E1313 was added but it seems at least E5000, E5001 are new as well. Are there more that were missing?

Comment (lucaCambi77) [reply src/developer/web-api/tracker.md:1141]: Ok, good catch; those ones were added in the code base but not in the docs, so I am adding those also here. So, to be precise, we are adding also what is missing from the current version @muilpp

Comment (teleivo) [reply src/developer/web-api/tracker.md:1141]: got it 👌🏻

**Review by teleivo — APPROVED** [review]

**Review by muilpp — APPROVED** [review]

**Pull request merged**